### DISCUSSION TITLE
Harish @ CapitalOne in North America(Boston)

### DISCUSSION THREAD

**Discussion by Harish110283** [Green-Software-Foundation/discussions#41] [Introductions]
Hi Everyone,

My name is Harish and I a software engineering manager at CapitalOne. I was alarmed by the climate impact when I visited Scotland last year and that led me to understand the science behind climate change. One thing led to another and here I am drawn towards how I can help create an impact by building sustainable software.

The materials in the foundation have helped me initiate "Green software" conversations within my company. I am hopeful to continue to learn from this community, perhaps contribute to discussion/build software and also continue to evangelize "Green software" principles in my current company.

You can find me on linked in here (https://www.linkedin.com/in/harish-ramamurthi).

Thanks
Harish

**Comment by NAMRATA-WOKE** [comment]
Hi Harish, welcome! It's great to hear your interest in sustainability and commitment to greening software. Have you taken the Green Software Practitioner course? [Green Software Practitioners](https://training.linuxfoundation.org/training/green-software-for-practitioners-lfc131/) is a free online educational course to help practitioners **meet a minimum and consistent level of understanding**. A certificate of completion is given as recognition from the Linux Foundation and GSF for the knowledge obtained to decarbonize software.

It takes about 2 hours to complete and its a course we're strongly recommending to practitioners who want to get more involved in the field and initiate green software projects at their respective companies.
